src/data_user.gd

```diff
@@ -12,6 +12,10 @@ signal connected
 signal disconnected
 
 func _ready() -> void:
+	for index1 in len(angles):
+		for index2 in len(angles[index1]):
+			angles[index1][index2] = (angles[index1][index2]/360)*2*PI
```

Comment:
Dit kan weg en in plaats daarvan kan je on the fly het vertalen naar radialen hieronder